lib/utils/image.rb

```diff
@@ -1,187 +1,45 @@
 require "tempfile"
 require "marcel"
 require "base64"
+require "vectory"
 
 module Metanorma
   module Utils
     class << self
-      class Namespace
```

Comment:
Please restore class Namespace, it is used elsewhere in the stack, independently of Vectory

Reply:
@alexeymorozov let’s move the Namespace module to a separate file.

Reply:
I restored it and moved to a separate class, but meanwhile realized it's very hard to access it - it's "hidden" in an eigenclass of `Metanorma::Utils`. So it could not be used like `Metanorma::Utils::Namespace` and I could not find any mention of it in the metanorma group. Just for information, we can keep it until I replace other code in other repositories with Vectory.